mssql-plugin/src/e2e-test/features/mssql/mssql sink/RunTime.feature

```diff
@@ -60,3 +60,55 @@ Feature: Mssql - Verify Mssql sink data transfer
     Then Close the pipeline logs
     Then Validate the values of records transferred to target MsSql table is equal to the values from source BigQuery table
 
+  @MSSQL_SOURCE @MSSQL_TARGET @CONNECTION @Mssql_Required
+  Scenario: To verify data is getting transferred from Mssql to Mssql successfully with use connection using MAX type datatypes
```

Comment:
what do we mean by MAX type datatypes?

Reply:
It includes Varchar(max),Varbinary(max),NVarchar(max) datatypes.